docs/sphinx/install.rst

```diff
@@ -71,4 +71,15 @@ Run the following command in your SSH terminal and follow the instructions
 
 .. code-block:: bash
 
-    cd; bash <(wget -qO- https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/future3/main/installation/install-jukebox.sh)
+    cd; bash <(wget -qO- https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/future3/develop/installation/install-jukebox.sh)
```

Comment:
Should that not be `main` instead of `develop`?

Reply:
This will always get the latest install-jukebox.sh, which *then* will get main or develop depending on the settings of GIT_BRANCH / GIT_USER. 
Need to be develop to get this new feature :-)